python/dynamic_graph/sot/torque_control/main.py

```diff
@@ -78,9 +78,9 @@ def main_v3(robot, startSoT=True, go_half_sitting=True, conf=None):
     connect_ctrl_manager(robot);
 
     # create low-pass filter for motor currents
-    robot.current_filter = create_butter_lp_filter_Wn_05_N_3('current_filter', dt, conf.motor_params.NJ)
```

Comment:
You should leave these 3 lines because this is the filtering I tested on the robot for the current.